src/Inspection.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+namespace RicorocksDigitalAgency\Morpher;
+
+
+class Inspection
+{
+    public $beforeMigratingHooks;
+    public $beforeHooks;
+    public $afterHooks;
+
+    public function __construct()
+    {
+        $this->beforeMigratingHooks = collect();
+        $this->beforeHooks = collect();
+        $this->afterHooks = collect();
+    }
+
+    public function beforeMigrating(callable $closure)
```

Comment:
Could we add return types to these methods for ide suggestion purposes?

Reply:
Added